config/controller-config-overlay.yml

```diff
@@ -0,0 +1,37 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@ def is_config_empty(config):
```

Comment:
I wanted to make sure that we don't make any changes to our releases. Is there a better way of checking if all the values are empty?